doc/release/1.4.0-notes.rst

```diff
@@ -73,6 +73,13 @@ lambda expressions for them.
 Deprecated features
 ===================
 
+`scipy` deprecations
+--------------------
+Support for NumPy functions exposed via the root SciPy namespace is deprecated
+and will be removed in 2.0.0. For example, if you use ``scipy.rand`` or
```

Comment:
Maybe add a reassuring sentence here:
`... 2.0.0. They remain available in the currently continuing Scipy 1.x release series.`

Should also be added here:
https://github.com/scipy/scipy/wiki/Release-note-entries-for-SciPy-1.4.0